notification-server/src/processor.rs

```diff
@@ -0,0 +1,73 @@
+use concordium_rust_sdk::{
```

Comment:
I want to test this. 

While that ended up being more cumbersome than thought of (https://concordium.slack.com/archives/C03DRCC9GR1/p1718886364935479) I would prefer doing that in another PR to lower the size of this PR.

That is ok with me as we are currently not using this yet.